pkgs/top-level/all-packages.nix

```diff
@@ -39535,6 +39535,8 @@ with pkgs;
 
   go-license-detector = callPackage ../development/tools/misc/go-license-detector { };
 
+  upspin = callPackage ../tools/filesystems/upspin { };
+
```

Comment:
Place in alphabetical order in the approximate position.

Reply:
Done.